bmv2/tests/openflow/Makefile.am

```diff
@@ -0,0 +1,38 @@
+if BMV2_TEST_ENV
+
+check_PROGRAMS = oftest
+
+oftest_SOURCES = oftest.c
+
+oftest_CPPFLAGS = -I$(includedir)
+oftest_CPPFLAGS += -I$(srcdir)/../thrift_endpoint
+oftest_CPPFLAGS += -I$(builddir)/../../oftest
+oftest_CPPFLAGS += -I$(top_srcdir)/pdfixed
+oftest_CPPFLAGS += -D_BMV2_
+
+if P4THRIFT
+    oftest_CPPFLAGS += -DP4THRIFT
+endif
+
+oftest_LDFLAGS = -L$(libdir)
+oftest_LDFLAGS += -pthread
+
+noinst_LTLIBRARIES = libthrift_endpoint.la
```

Comment:
Could you put it in the `Makefile.am` of the parent directory? This way both tests can use `libthrift_endpoint.la` in the same way.